pkg/3scale/amp/operator/upgrade.go

```diff
@@ -53,6 +54,14 @@ func (u *UpgradeApiManager) Upgrade() (reconcile.Result, error) {
 		return res, nil
 	}
 
+	res, err = u.upgradePodTemplateLabels()
```

Comment:
An obsevation about this: This means that from now on, for every new operator release where any of this label changes all the pods in 3scale will be redeployed.

Reply:
That is correct